appengine/expiration.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""Delete expired XML.
+"""
+
+__author__ = "fenichel@google.com (Rachel Fenichel)"
+
+
+import storage
+import datetime
+
+from google.cloud import ndb
+
+def delete_expired():
+  # Delete entries that have not been accessed in more than a year.
```

Comment:
This seems like it's meant to be a method comment. In which case, you should use the format for docstring:
`"""Delete entries that have not been accessed in more than a year."""`

Related nit: Deletes instead of Delete (coming from style guide for wording)

Reply:
Done.